testdata/p4_16_samples_outputs/issue3274-1-FrontEnd_41_SimplifyDefUse-first.p4

```diff
@@ -0,0 +1,12 @@
+void x() {
+}
```

Comment:
Should these expected output files with "FrontEnd_41_SimplifyDefUse" be in this PR?  I don't see any input P4 program in the p4c repo that corresponds to them.  Perhaps added into this PR by mistake due to some local new test P4 program you have?

Reply:
I have deleted these.
In principle gauntlet checks equivalence of the code, so it should flag any file where the semantics changes.
But it does not check all files, and not between all compilation points.
@fruffy what are the representations checked? We should document this somewhere.
I have reviewed some -frontend files, but not all.

Reply:
Gauntlet currently only checks p4test's front and mid end. I have also disabled parser checking for Gauntlet (the missing implementation for extract causes issues). The only pass that is skipped is [FlattenHeaderUnion](https://github.com/p4gauntlet/toz3/blob/master/compare/compare.cpp#L14). I can try to add a README on this.